sdk/template/azure-sdk-template/README-EXAMPLE.md

```diff
@@ -0,0 +1,125 @@
+# Azure Template client library for Java
```

Comment:
Why do you have README-EXAMPLE and README? shouldn't we just have README?

Reply:
I'll merge them together, I added `README-EXAMPLE` since the `README` had a link to `README-EXAMPLE` that was dead.